core/src/test/scala/integration/kafka/server/ReplicaFetcherThreadFatalErrorTest.scala

```diff
@@ -45,7 +45,10 @@ class ReplicaFetcherThreadFatalErrorTest extends ZooKeeperTestHarness {
   @After
   override def tearDown() {
     Exit.resetExitProcedure()
-    brokers.foreach(_.shutdown())
+    brokers.par.foreach { s =>
+      s.shutdown()
+      CoreUtils.delete(s.config.logDirs)
+    }
```

Comment:
Is it worth having a utility methods that takes a sequence of brokers and does all of this?